core/config/config.go

```diff
@@ -212,6 +212,25 @@ type DatabaseConfig struct {
 	// If not set or value can't be parsed, defaults to "serializable".
 	// Intention is to migrate to "read committed" for performance reasons after testing is complete.
 	IsolationLevel string
+
+	// MigrateStreamCreation indicates to the database that all new streams should be allocated
+	// in one of the 256 pre-allocated stream partitions for either regular or media streams instead
+	// of allocating new tables per stream according to the legacy schema. If this flag is unset, a
+	// node will continue to allocate new tables for each stream as it is created.
+	MigrateStreamCreation bool
+
+	// NumPartitions specifies the number of partitions to use when creating the schema for stream
+	// data storage. If <= 0, a default value of 256 will be used. No more than 256 partitions is
+	// supported at this time.
+	NumPartitions int
+}
+
+func (c *DatabaseConfig) Init() error {
```

Comment:
No need for this method, just set this setting to 256 in GetDefaultConfig

Reply:
We don't support > 256 partitions, does it make sense to impose an upper bound here?